### PR TITLE
feat: update aquaculture model

### PR DESCRIPTION
**Pull request by AHABID**

### CODE REVIEW

File: Specie/schema.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (albertoabellagarcia) [inline Specie/schema.json:1]: There are no definitions.. ¿¿??

File: Specie/examples/example-normalized.jsonld [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (albertoabellagarcia) [inline Specie/examples/example-normalized.jsonld:1]: It does not look like a real payload.

File: Sump/schema.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (albertoabellagarcia) [inline Sump/schema.json:1]: please the definitions could be more precise (indeed some of the could be available already in the search database https://smartdatamodels.org/index.php/ddbb-of-properties-descriptions/
Review them

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
please address the changes requested

File: Specie/examples/example-normalized.jsonld (hunk 2 above)
Comment (AHABID) [reply Specie/examples/example-normalized.jsonld:1]: fixed

**Comment by albertoabellagarcia** [comment]
This modification is obsolete. Please reopen when there are actual contribution.

**Pull request closed without merging**